Sources/TestHelper/DummyRoute.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import MapboxCoreNavigation
+import MapboxDirections
+
+public class DummyRoute: Route {
+
+    public init(description: String) {
+        let leg = RouteLeg(steps: [],
+                           name: description,
+                           distance: 0,
+                           expectedTravelTime: 0,
+                           typicalTravelTime: nil,
+                           profileIdentifier: .automobile)
+        super.init(legs: [leg], shape: nil, distance: 0, expectedTravelTime: 0, typicalTravelTime: nil)
```

Comment:
Little hack, because `description` property cannot be overriden

Reply:
This is correct, since `Route.description` is based on `RouteLeg.name`:

https://github.com/mapbox/mapbox-directions-swift/blob/28a0bc98ca13ac7cbfa08d06f273dcd94503d696/Sources/MapboxDirections/DirectionsResult.swift#L191
https://github.com/mapbox/mapbox-directions-swift/blob/28a0bc98ca13ac7cbfa08d06f273dcd94503d696/Sources/MapboxDirections/RouteLeg.swift#L353